common/stats/disk_monitor.go

```diff
@@ -0,0 +1,97 @@
+package stats
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// DiskMonitor monitor disk usage for selected directories
+type DiskMonitor struct {
+	pathAbbrevs []string // these abbreviations are added to the end of the paths' usage gauge names
```

Comment:
I hate to nit naming but pathAbbrevs isn't clear without reading the comment, suggest something like statSuffix